doc/src/appendices/suiterc-config-ref.rst

```diff
@@ -1257,7 +1257,7 @@ represents a group or family of tasks if other namespaces inherit from
 it, or a task if no others inherit from it.
 
 .. todo::
-   follolwing para here is a '%' thing i.e. not ment to be seen /comment:
+   following para here is a '%' thing i.e. not meant to be seen /comment:
```

Comment:
Oh wow - well done for spotting this! There are multiple typos in this line because I only added it (typing very rapidly because it just needed to be readable) as a reminder to myself when I went through & converted, incrementally by feature, the LaTeX content to Sphinx in #2910, & clearly forgot to remove it.

It is a meaningless comment now so please just remove the whole line & the ``.. todo::`` directive it is in.

Reply:
I pushed a commit to delete the comment.